src/QirRuntime/test/FullstateSimulator/FullstateSimulatorTests.cpp

```diff
@@ -197,7 +199,7 @@ TEST_CASE("Fullstate simulator: TTSAdj=Id", "[fullstate_simulator]")
         iqa->H(q);

```

Comment:
I like these tests! Perhaps it's worth adding tests for adjoint R and adjoint Exp to verify the angle flipping now that those operations are present?

Reply:
The angles bugs in this PR were at a different level and wouldn't be caught by these tests, but it does add coverage to provide these tests.